Lesson17/css/style.css

```diff
@@ -134,3 +134,18 @@ img {
   top: -5px;
   transform: rotate(40deg);
 }
+
+.arrow-btn:disabled {
+  background-color: rgb(170, 170, 170);
+}
+
+.count-wrapper {
+  display: flex;
+  align-items: center;
+  font-size: 20px;
+}
+
+.total-number::before {
+  content: "/";
+  margin: 0 5px;
+}
```

Comment:
IMO
On the screen visible to the user, it shows 1/5, but if you check the html, you will see that `/` has become a pseudo-element and it shows `<p>1</p><p>5</p>`. From the viewpoint of accessibility, I think this display is undesirable.
It would be better if it could be implemented in such a way that it is clear that the first of the five pages is displayed.

Reply:
Thanks for the review.

The tags have been changed to span, and the diagonal lines have been inserted as text. Also, the class name has been changed.
Please check it out.
85adf4e

Reply:
It seems to be showing `1//5`.
<img width="1071" alt="スクリーンショット 2021-12-10 18 45 03" src="https://user-images.githubusercontent.com/43788628/145553239-7f816822-cb45-428a-bd8a-f3063ab8c530.png">

Reply:
Thank you!
Updated [codesandbox](https://codesandbox.io/s/js-lesson17-part2-mznov).